Zappy_IA_Report/DTC/Code/data.py

```
@@ -0,0 +1,62 @@
+DATASET_FILE = sys.argv[1]
+MODEL_FILE = sys.argv[2]
+FEATURES = [
+    "food", "linemate", "sibur", "deraumere", "mendiane",
+    "phiras", "thystame", "level", "players_nearby"
+]
+model = None
+
+def load_data() -> pd.DataFrame:
+    if os.path.exists(DATASET_FILE):
+        return pd.read_csv(DATASET_FILE)
+    return pd.DataFrame(columns=FEATURES + ["actions"])
```

Ruff (0.11.9)

Consider `[*FEATURES, "actions"]` instead of concatenation

Replace with `[*FEATURES, "actions"]`

(RUF005)